snap/info_snap_yaml.go

```diff
@@ -355,7 +355,7 @@ func setAppsFromSnapYaml(y snapYaml, snap *Info, strk *scopedTracker) error {
 			PostStopCommand: yApp.PostStopCommand,
 			RestartCond:     yApp.RestartCond,
 			RestartDelay:    yApp.RestartDelay,
-			BusName:         yApp.BusName,
+			ActivatesOn:     yApp.ActivatesOn,
```

Comment:
I've asked in the [forum](https://forum.snapcraft.io/t/support-for-daemon-dbus/8855/11) for approval of the full syntax and if bus-name is meant to be immediately obsoleted.

Reply:
It looks like this is still pending

Reply:
if Info.ActivatesOn is a []*SlotInfo then this should be probably be setup close to where we set AppInfo.Slots.

As we discussed we should make activates-on imply these slots as being listed as if they were in the "slots" field for the app. Repetition between the two should be ok though.